source/install/self-managed-changelog.md

```diff
@@ -6,12 +6,122 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
+
+**IMPORTANT:** If you upgrade from a release earlier than v7.8, please read the other [Important Upgrade Notes](/upgrade/important-upgrade-notes.html).
+
+### Compatibility
+ - Updated Firefox minimum supported version to 102+.
+ - Updated Safari minimum supported version to 16.2+.
+ - Updated Windows minimum supported version to 10+.
+ - Updated Chromium minimum supported version to 110+.
+ - Updated Edge minimum supported version to 110+.
+
+### Highlights
+
+#### Calls
+ - 
+
+#### Boards
+ - System and team admins are now able to join any board on the team as a board admin via the board URL.
+ - Additional Compliance APIs to return the history of boards and blocks, including deleted items (available in Mattermost Enterprise Edition and above).
+ 
+#### Playbooks
+ - 
+
+### Improvements
+
+#### User Interface (UI)
+ - Added use case templates to help users quickly and easily set up all the tools they need to accomplish their work by use cases. Each use case template contains a channel, a Board and/or a playbook, and integrations.
+ - All post components were removed in favor of a unified approach.
+ - App bindings are now refreshed when a App plugin-enabled event gets triggered.
+ - Improvements were added to the sidebar channel and category menus.
+ - Removed right-click hijacking on code blocks in messages.
+ - The order of the Leave Channel and Archive Channel settings were updated to match the mobile app.
+ - Added the condition to remove unread styling for archived channels and to filter archived channels from local data.
+ - Changed the collapsed post fade out effect to be less buggy.
+ - Users now have the ability to see the history of edited messages and to restore an old message version with the current version.
+ - Improved the user interface of the user profile popover.
+
+#### Administration
+ - Boards cards are no longer mentioned as being limited in the **System Console**, the limits usage modal, the downgrade modal, or the left-hand side menu.
+ - Removed unused ``ProductLimits.Integrations``.
+ - Export files now contain the read and unread status for channels.
+ - While upgrading a High Availability installation with rolling upgrades, after upgrading all nodes, the System Console may display a single disabled plugin. This is expected due to the way plugins are initialized. Prepackaged plugins are copied over to the plugins directory. Following another restart of the first node upgraded, the System Console should no longer show any plugins present or enabled.
+ - Added an error message when running an LDAP sync with ``SyncEnabled`` set to ``false``.
+ - Added Admin log table filtering and sorting.
+ - GraphQL APIs are now correctly counted when measuring performance telemetry.
+ - Added a dynamic call-to-action under the **System Console > True Up Review System** subsection for air-gapped and non-air-gapped systems.
```

Comment:
Actually, the true-up CTA appears on System Console > Site Statistics and > Team Statistics

The banner reminding them about true-up follows the schedule outlined here:
https://docs.mattermost.com/about/self-hosted-subscriptions.html#quarterly-true-up-report